pkg/storage/factory.go

```diff
@@ -295,7 +295,7 @@ type Config struct {
 	DisableBroadIndexQueries bool         `yaml:"disable_broad_index_queries"`
 	MaxParallelGetChunk      int          `yaml:"max_parallel_get_chunk"`
 
-	UseThanosObjstore bool          `yaml:"use_thanos_objstore" doc:"hidden`
+	UseThanosObjstore bool          `yaml:"use_thanos_objstore" doc:"hidden"`
```

Comment:
I'm confused how this is related?

Reply:
it's not, my editor corrected the incorrect tag

Reply:
i think i'll keep this change here instead of raising another pr for this small correction :)